src/Components/ADT3DSceneBuilder/Internal/Behaviors/Widgets/WidgetBuilders/DataHistoryWidgetBuilder/Internal/TimeSeriesFormCallout.tsx

```diff
@@ -0,0 +1,173 @@
+import React, {
```

Comment:
When there are 3 series, editing one causes it to be anchored at the top of the form, not on the thing you clicked on.  a bit confusing IMO

Reply:
couldn't reproduce it, can you send me screen recording? When there are 3 series, editing one didn't cause it to be anchored at the top of the form?
![3TimeSeries_onedit](https://user-images.githubusercontent.com/45217314/194952303-e22cfef8-5eed-4156-a47c-357bdc6b5e55.gif)

Reply:
that anchor doesn't seem to be right even in your recording here. It should be next tot he item you clicked and not at the top of the screen right?

Reply:
edit on overflow menu and onclick of list item tied to the same callback. not sure which anchor are you both referring to?

Reply:
The callout should open to the right of the list item.

Reply:
oh okay you mention the callout target, yeah it is set to the "Add time series" action button (callout appears at the bottom of that action button target). but since when there are 3 series I am hiding that action button that is why it was anchoring to the top then. let me see if I can improve it.
![image](https://user-images.githubusercontent.com/45217314/194957740-48f341da-0218-42c1-bd80-69898fd07743.png)

Reply:
edit callout is anchored to the bottom of the list item

Reply:
looks great now